src/dodal/devices/oav/pin_image_recognition/__init__.py

```diff
@@ -151,27 +134,32 @@ async def connect(self, sim: bool = False):
         await self.preprocess_iterations.set(5)
         await self.preprocess_ksize.set(5)
 
-    async def read(self) -> dict[str, Reading]:
-        tip_pos, timestamp = await asyncio.wait_for(
-            self._get_tip_position(), timeout=await self.timeout.get_value()
-        )
-
-        return OrderedDict(
-            [
-                (self._name, {"value": tip_pos, "timestamp": timestamp}),
-            ]
-        )
+    @AsyncStatus.wrap
+    async def trigger(self):
+        async def _set_triggered_tip():
+            """Monitors the camera data and updates the triggered_tip signal.
+
+            If a tip is found it will update the signal and stop monitoring
+            If no tip is found it will retry with the next monitored value
```

Comment:
As discussed I'll add a comment suggesting to add this to ophyd_async docs as a good example of `observe_value`